### PR TITLE
Adding the project as a template to work with react-native-community/cli

### PR DESCRIPTION
**Pull request by AbdoHema2016**
Changing the projects directory structure and adding necessary files to create a project using the react-native-community/cli

### CODE REVIEW

File: template/InnovatorRN/_env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+IS_PRODUCTION=true
```
Comment (ajaykumar97) [inline template/InnovatorRN/_env:1]: The env vars are supposed to be private and not pushed to the VSC. Please remove the `_env` file.

File: template/InnovatorRN/_env.production [hunk 2]
```diff
@@ -0,0 +1,2 @@
+IS_PRODUCTION=true
```
Comment (ajaykumar97) [inline template/InnovatorRN/_env.production:1]: The env vars are supposed to be private. Please remove the `_env.production` file.

File: template/InnovatorRN/_env.staging [hunk 3]
```diff
@@ -0,0 +1,2 @@
+IS_PRODUCTION=true
```
Comment (ajaykumar97) [inline template/InnovatorRN/_env.staging:1]: The env vars are supposed to be private. Please remove the `_env.staging` file.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "starter",
+    "react-native",
+    "react",
+    "native"
+  ],
+  "bugs": {
+    "url": "git@github.com:uCreateit/react-native-innovator.git/issues"
+  },
+  "dependencies": {
+    "react-native-localization": "^2.1.6"
```
Comment (ajaykumar97) [inline package.json:24]: `react-native-localization` is not needed. Please remove the `dependencies` section from this root level `package.json` file.

**Review by ajaykumar97 — CHANGES_REQUESTED** [review]

File: template/InnovatorRN/_env (hunk 1 above)
Comment (AbdoHema2016) [reply template/InnovatorRN/_env:1]: its still not pushed once the app is created they would change into a .env and they r already added on the git ignore, adding them here provide a ready to go example for developer using it

File: template/InnovatorRN/_env.production (hunk 2 above)
Comment (AbdoHema2016) [reply template/InnovatorRN/_env.production:1]: its still not pushed once the app is created they would change into a .env and they r already added on the git ignore, adding them here provide a ready to go example for developer using it

Comment (ajaykumar97) [reply template/InnovatorRN/_env.production:1]: Yeah, makes sense. Ultimately these files will be converted into `.env` files and will be ignored as they are added into the `.gitignore`.

**Pull request merged**